libs/core/scripts/build-tokens.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs-extra';
+import * as tokens from '../tokens/tokens.json';
+
+/**
+ * 
+ * @param tokens Tokens data in JSON format.
+ * @param base Base string to format custom properties.
+ * @returns Formatted CSS with all custom properties.
+ */
+const tokensToCss = (tokens: object, base = `-`) =>
+  Object.entries(tokens).reduce((css, [key, value]) => {
+		let newBase = base + `-${key}`;
+		if (!['description', 'type'].includes(key)) {
+			newBase = newBase.replace(/\-value/, '');
+			if (typeof value !== "object") {
+				return css + newBase + `: ${value};\n`
+			}
+			return css + tokensToCss(value, newBase)
+		} 
+		return css
+  }, ``);
+
+/**
+ * Reads the index.scss file if it exists and adds an import statement per category name.
+ * If the file doesn't exist, it will be created.
+ * 
+ * @param categoryName The name of the token category.
+ */
+const formatIndexFile = (categoryName: string) => {
+	const indexFile = `${process.cwd()}/src/global/styles/tokens/index.scss`;
+	const importLine = `@use '${categoryName}';\n`;
+	if (!fs.existsSync(indexFile)) {
+		fs.writeFileSync(indexFile, importLine);
+	} else {
+		fs.readFile(indexFile, (err, data) => {
```

Comment:
minor suggestion...You can use `fs.readFileSync` to be consistent with the other uses of this method.